pkg/epp/requestcontrol/director.go

```diff
@@ -278,16 +291,45 @@ func (d *Director) toSchedulerPodMetrics(pods []backendmetrics.PodMetrics) []sch
 	return pm
 }
 
-func (d *Director) HandleResponse(ctx context.Context, reqCtx *handlers.RequestContext) (*handlers.RequestContext, error) {
+// HandleResponseReceived is called when the first chunk of the response arrives.
```

Comment:
`HandleResponseReceived is called when the first chunk of the response arrives.`
as far as I know, this is incorrect and this function is called when response headers are received, not the first chunk. I also recommend to rename the function to specify that exactly to something like:
`HandleResponseHeaders`
this godoc comment should be fixed to something like:
```suggestion
// HandleResponseHeaders is called when the response headers are received.
```

Reply:
done